src/buildercore/bootstrap.py

```diff
@@ -162,6 +166,18 @@ def is_resourcing():
 
     stack_all_ec2_nodes(stackname, _setup_ec2_node, username=BOOTSTRAP_USER)
 
+def setup_terraform(stackname, context):
+    if not context.get('fastly'):
```

Comment:
these are feature flags that prevent any stack not using Fastly from going through this unstable code for now